modules/openy_gc_auth/modules/openy_gc_auth_yusa/src/Plugin/GCIdentityProvider/YUSA.php

```diff
@@ -105,12 +105,14 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
 
     $form['association_number'] = [
       '#title' => $this->t('Association number'),
-      '#type' => 'number',
-      '#min' => 0,
-      '#max' => 9999,
-      '#size' => 4,
+      '#type' => 'textfield',
       '#description' => $this->t('This is usually 4 digits.'),
       '#default_value' => $config['association_number'],
+      '#attributes' => [
+        'inputmode' => 'numeric',
+        'pattern' => '[0-9]*',
```

Comment:
@sunil-ymca this should be `'pattern' => '[0-9]{4}'` and then I think we'll be good to go.

Reply:
@froboy: I have just pushed association field pattern changes ('pattern' => '[0-9]{4}'), kindly check and review it.